### ISSUE TITLE
Unable to deply any application via Azure extension

### ISSUE THREAD

**Issue by sidchiku9** [microsoft/vscode-azureappservice#2199]
Issue Type: <b>Bug</b>

Log in to the Azure Extension
Click on the cloud icon that is present in the app service and select any web app that you want to deploy.
It will run successfully till creating the ZIP file but after that it gives an ECONNRESET.
Please look into it as with older versions I was able to execute the same.

Extension version: 0.23.1
VS Code version: Code 1.66.1 (8dfae7a5cd50421d10cd99cb873990460525a898, 2022-04-06T14:49:47.400Z)
OS version: Darwin x64 21.4.0
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8257U CPU @ 1.40GHz (8 x 1400)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|3, 3, 3|
|Memory (System)|8.00GB (0.30GB free)|
|Process Argv|--crash-reporter-id bee57bbb-8f97-48f8-95c4-78b4b14345ca|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes516:30244333
pythonvspyl392:30443607
pythontb:30283811
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscorecescf:30445987
pythondataviewer:30285071
vscod805cf:30301675
pythonvspyt200:30340761
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
vsc1dst:30438360
pythonvs932:30410667
wslgetstarted:30449410
pythonvsnew555:30457759
vscscmwlcmt:30465135
cppdebugcf:30466690
pynewfile477:30463512

```

</details>

**Comment by alexweininger** [comment]
Until a fix is released in VS Code stable (hopefully the next major release), here are some options:
1. Wait for the fix to come in [VS Code Insiders](https://code.visualstudio.com/insiders/)
2. [Deploy via Azure CLI](https://github.com/microsoft/vscode-azurefunctions/issues/2844#issuecomment-1086259916)
3. Downgrade your Azure App Service extension version to `v0.21.1` (Mar 04, 2021), we used a different zip deploy method which seems to be working still. Note: it might not work in environments like WSL or Codespaces, which is why we changed our deploy in `v0.21.2` (Mar 09, 2021).
4. Downgrade to the [VS Code February 2022 release](https://code.visualstudio.com/updates/v1_65), where zip deploy is working as discussed and confirmed above.

See [this issue](https://github.com/microsoft/vscode/issues/146470) for more information.
